src/components/PageLayout.js

```diff
@@ -33,43 +39,37 @@ export class Psegment extends React.Component {
 }
 
 Psegment.defaultProps = {
-    style: { padding: "2em 2em" },
+    style: { margin: "0 auto", padding: "2em 2em", maxWidth: theme.pageSize.maxSize },
     vertical: true
 };
 
 export class Pgrid extends React.Component {
     render() {
         const { children, ...other } = this.props;
-        return <Grid {...other}>{children}</Grid>;
+        return <div {...other}>{children}</div>;
     }
 }
 
-Pgrid.defaultProps = {
-    stackable: true,
-    container: true
-};
-
 export class Pcolumn extends React.Component {
     render() {
         const { children, ...other } = this.props;
-        return <Grid.Column {...other}>{children}</Grid.Column>;
+        return <Grid.Unit {...other}>{children}</Grid.Unit>;
     }
 }
 
 Pcolumn.defaultProps = {
     style: {
-        paddingBottom: "1em",
-        paddingTop: "1em"
+        padding: "1rem"
```

Comment:
do you need `padding-top`?

Reply:
I set total padding so I do not understand your question.
Can you explain it?